doc/man3/EVP_EncryptInit.pod

```diff
@@ -479,7 +479,9 @@ The following I<ctrl>s are supported for the ChaCha20-Poly1305 AEAD algorithm.
 
 Sets the nonce length. This call can only be made before specifying the nonce.
 If not called a default nonce length of 12 (i.e. 96 bits) is used. The maximum
-nonce length is 16 (B<CHACHA_CTR_SIZE>, i.e. 128-bits).
+nonce length is 12 bytes (i.e. 96-bits). If a nonce of less than 12 bytes is set
+then the nonce is automatically front padded with 0-bytes until the nonce is 12
```

Comment:
Will everyone understand "front padded"? I'm wondering if that might need a slight rephrase...

Reply:
Hmm, "front padded" makes sense to me. I could talk about the "leading bytes" being set to 0 if you prefer, but I'm not sure if it's any clearer. Unless you have a suggestion?

Reply:
"then the nonce is automatically padded with leading 0 bytes to make a 12 byte nonce"

Reply:
Fixup commit pushed including that wording.